test/jdk/java/util/concurrent/ThreadPerTaskExecutor/ThreadPerTaskExecutorTest.java

```diff
@@ -603,20 +572,19 @@ void testInvokeAnyWithInterruptSet(ExecutorService executor) throws Exception {
     @MethodSource("executors")
     void testInterruptInvokeAny(ExecutorService executor) throws Exception {
         try (executor) {
-            Callable<String> task1 = () -> {
-                Thread.sleep(Duration.ofMinutes(1));
-                return "foo";
-            };
-            Callable<String> task2 = () -> {
-                Thread.sleep(Duration.ofMinutes(2));
-                return "bar";
-            };
-            scheduleInterrupt(Thread.currentThread(), Duration.ofMillis(500));
+            var task = new LongRunningTask<Void>();
             try {
-                executor.invokeAny(Set.of(task1, task2));
+                scheduleInterruptAt("java.util.concurrent.ThreadPerTaskExecutor.invokeAny");
+                executor.invokeAny(Set.of(task));
```

Comment:
Seems strange to test invokeAny when there is only one task?

Reply:
> Seems strange to test invokeAny when there is only one task?

There are several tests further up that exercise invokeAny with more than one task. This one is testing interrupt of invokeAny, we could expect it to test interrupting with many tasks if needed.

Reply:
OK - looks fine to me then.